File: ruby/tests/700-index-tasks.rb
normal         = normal_handle
@task_list_700 = normal.index_tasks()

puts "LIST (first tasks)= \n#{show_top_entries(@task_list_700,5)}" if @debug

raise "Value returned is not an array" unless @task_list_700.present? && @task_list_700.is_a?(Array)

true

File: perl/tests/700-index-tasks.pl
my $normal         = &NormalHandle;
our $task_list_700 = $normal->index_tasks();

if ($DEBUG) {
  my $top_entries = &ShowTopEntries($task_list_700,4);
  print "LIST (first tasks)=\n${top_entries}\n";
}

die "Value returned not an array.\n" unless defined($task_list_700) && ref($task_list_700) eq "ARRAY";

1;
